### ISSUE TITLE
Windows Installer for Twisted 8

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3115]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| moonfallen reported|
|-|-|
|Trac ID|trac#3115|
|Type|enhancement|
|Created|2008-03-23 21:45:20Z|

Twisted 8 has yet another tarball structure.  Update the Windows Build stuff to use it.

Attachments:

* [3115.txt](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/313/313a97d1d2596eaaba2fda4cf3233412ede80197/ade5ba23b0bc68f7c9c5bf108089a05c8cbe3ede.txt) (3040 bytes) - added by moonfallen on 2008-03-24 00:35:00Z - Update win32 builder for Twisted 8

<details><summary>Searchable metadata</summary>

```
trac-id__3115 3115
type__enhancement enhancement
reporter__moonfallen moonfallen
priority__normal normal
milestone__twisted_8_0 twisted-8.0
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__patch patch
time__1206308720000000 1206308720000000
changetime__1206845230000000 1206845230000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen set owner to moonfallen|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|moonfallen set owner to @radix|
|-|-|

-> radix for review and commit.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_5"></a>|@radix set owner to moonfallen|
|-|-|

I guess you want to get rid of python2.3 support? That sounds fine to me, I just want to make sure that's what's going on.

Can you explain what the following does?

+    :sys $(pyexe) setup.py build --compiler=mingw32 \
+                                install --root=win32dist --no-compile

I realize a similar line was already there, I just don't think I can evaluate changes to it without understanding it. It looks like a strange setup.py command line, but maybe that's not one single shell command...

Get back to me and I'll merge it to trunk.

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|moonfallen commented|
|-|-|

Like 'make', setup.py lets you put multiple targets on the same line.  This is equivalent to

python setup.py build -c mingw32
python seutp.py install --root=win32dist --no-compile

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|moonfallen commented|
|-|-|

er, damn wiki formatting.  

python setup.py build -c mingw32

python setup.py install --root=win32dist --no-compile

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_8"></a>|@radix set status to `closed`|
|-|-|

(In [23132]) Apply patch from Cory Dodt for resolution of [[#3115](https://github.com/twisted/twisted/issues/3115)](https://github.com/twisted/twisted/issues/3115).

Author: moonfallen
Review: radix
Fixes #3115

This brings the AAP build files up to date with the new Twisted release
format. It also drops support for python 2.3, adds support for python 2.5,
and updates the name of the zope interface package.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
